daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -570,7 +570,7 @@ enum BuiltinFunction {
   NUMERIC_TO_BIGNUMERIC = 145; // *Available in versions >= 1.13*
   BIGNUMERIC_TO_TEXT = 146; // *Available in versions >= 1.13*
 
-  TYPEREP_TYCON_NAME = 148; // *Available in versions >= 1.dev*
+  TYPEREP_TYCON_NAME = 148; // *Available in versions >= 1.15*
```

Comment:
I couldn't find any uses of this in v1.15, but I did notice the daml code for it uses `#ifdef DAML_EXPERIMENTAL` when it should have its own feature flag (I'll take care of that in a separate PR).

https://github.com/digital-asset/daml/blob/1a58e9aa232b5f23533842a955724ebb0895999c/compiler/damlc/daml-stdlib-src/DA/Internal/Template/Functions.daml#L291-L304

Reply:
that is,

```suggestion
  TYPEREP_TYCON_NAME = 148; // *Available in versions >= 1.dev*
```

Reply:
The actual import piece is in [DecodeV1.scala](https://github.com/digital-asset/daml/blob/3abbf2b9725b45dbbf054de5bea35fd21a5d284d/daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala#L2397)
```scala
      BuiltinFunctionInfo(ANY_EXCEPTION_MESSAGE, BAnyExceptionMessage, minVersion = exceptions),
      BuiltinFunctionInfo(TYPEREP_TYCON_NAME, BTypeRepTyConName, minVersion = LV.v1_dev),
      BuiltinFunctionInfo(TEXT_TO_UPPER, BTextToUpper, minVersion = unstable),
```

by the way it should be with `TYPE_REP` to be consistent with the rest of the proto.

```suggestion
  TYPE_REP_TYCON_NAME = 148; // *Available in versions >= 1.dev*
```